ProcessLib/ProcessVariable.cpp

```diff
@@ -34,6 +34,9 @@ ProcessVariable::ProcessVariable(
       _bc_builder(new BoundaryConditionBuilder())
 {
     DBUG("Constructing process variable %s", _name.c_str());
+    //! \ogs_file_param{prj__process_variables__process_variable__order}
+    auto opt_shapefunction_order = config.getConfigParameterOptional<int>("order");
+    _shapefunction_order = (opt_shapefunction_order ? opt_shapefunction_order.get() : 1);
```

Comment:
it's not possible because Mesh/Element class doesn't know which order it is. In current implementation, LocalDataInitializer is checking if a proper element is found for the given shape function order.